llmproxy/config/internal_config.py

```diff
@@ -169,7 +169,12 @@
                 "name": "text-bison",
                 "cost_per_token_input": 1.5e-06,
                 "cost_per_token_output": 2e-06,
-            }
+            },
+            {
+                "name": "gemini-pro",
+                "cost_per_token_input": 0,
+                "cost_per_token_output": 0,
```

Comment:
Why is this zero? You need to make sure that the cost is per token. (In the case of text-bison they specifced that 1 character == 1 metric of cost)

![image](https://github.com/llm-proxy/llm-proxy/assets/46412618/eee5c6f2-93f0-4086-b40f-9022ab72b399)

Reply:
placeholder values for now. pr is a wip